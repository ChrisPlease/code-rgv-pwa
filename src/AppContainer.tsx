/**
 * Applicaion Container
 */
import * as React from "react";

export class AppContainer extends React.Component<any, any> {
  public render() {
    return (
      <div>
        <div className="container">
          {this.props.children}
        </div>
      </div>
    );
  }
}
