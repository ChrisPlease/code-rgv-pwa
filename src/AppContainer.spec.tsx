/**
 * AppContainer test
 */
import { MemoryRouter } from "react-router-dom";
import { shallow } from "enzyme";
import * as renderer from "react-test-renderer";
import * as React from "react";
import { AppContainer } from "./AppContainer";

test("AppContainer snapshot", () => {
  const component = renderer.create(
    <MemoryRouter>
      <AppContainer />
    </MemoryRouter>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("AppContainer", () => {
  const component = shallow(<AppContainer />);
  expect(component.find("div").length).toEqual(2);
});
