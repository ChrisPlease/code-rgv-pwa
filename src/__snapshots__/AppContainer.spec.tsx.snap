// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer exists 1`] = `
<div>
  <nav
    className="navbar"
    style={
      Object {
        "backgroundColor": "#f5f5f5",
        "border": "1px solid #ffde49",
      }
    }
  >
    <div
      className="navbar-brand"
    >
      <a
        className="navbar-item"
      >
        <img
          alt="brownsville logo"
          src="test-file-stub"
        />
        <span
          className="title is-4"
        >
          <strong>
            CodeRGV
          </strong>
        </span>
      </a>
      <div
        className="navbar-burger"
        data-target="codeRgvMenu"
      >
        <span />
        <span />
        <span />
      </div>
    </div>
    <div
      className="navbar-menu"
      id="codeRgvMenu"
    >
      <div
        className="navbar-start"
      >
        <a
          className="navbar-item"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
        <a
          className="navbar-item"
          href="/events"
          onClick={[Function]}
        >
          Events
        </a>
        <a
          className="navbar-item"
          href="/projects"
          onClick={[Function]}
        >
          Projects
        </a>
        <a
          className="navbar-item"
          href="/about"
          onClick={[Function]}
        >
          About
        </a>
      </div>
      <div
        className="navbar-end"
      >
        <a
          className="navbar-item"
          href="https://github.com/codergvbrownsville"
          rel="noopener noreferrer"
          role="button"
          target="_blank"
        >
          <i
            className="fa fa-github"
          />
        </a>
        <a
          className="navbar-item"
          href="https://www.facebook.com/codergvbrownsville/"
          rel="noopener noreferrer"
          role="button"
          target="_blank"
        >
          <i
            aria-hidden="true"
            className="fa fa-facebook-square"
          />
        </a>
      </div>
    </div>
  </nav>
  <div
    className="container"
  />
</div>
`;
